mon-pix/app/utils/session-storage-entry.js

```diff
@@ -0,0 +1,18 @@
+export class SessionStorageEntry {
+  constructor(key) {
+    this.key = key;
+  }
+
+  set(value) {
+    sessionStorage.setItem(this.key, JSON.stringify({ value }));
+  }
+
+  get() {
+    const result = JSON.parse(sessionStorage.getItem(this.key) || '{}');
+    return result?.value;
```

Comment:
L'opérateur `?.` me parait inutile : 

```suggestion
    return result.value;
```